app/controllers/application_controller.rb

```diff
@@ -1,6 +1,2 @@
 class ApplicationController < ActionController::Base
```

Comment:
- [ ] Kindly add the `authenticate_user!` method to the application controller so that only logged-in users can use the app.

  ```
  before_action :authenticate_user!
  ```